corehq/apps/linked_domain/templates/linked_domain/domain_links.html

```diff
@@ -23,144 +23,156 @@
+                        <i class="fa fa-times"></i> {% trans "Error" %}
+                      </button>
+                      <span class="help-block" data-bind="visible: error, text: error"></span>
+                      <a data-bind="visible: showUpdate && update_url, attr: {href: update_url}">{% trans "Go to update page" %}</a>
+                    </td>
+                  </tr>
+                  </tbody>
+                </table>
+              </div>
             </div>
           </div>
         </div>
       {% endif %}
-      <div class="tab-pane fade{% if not is_linked_domain %} in active{% endif %}" id="tabs-linked">
-        <div data-bind="if: linked_domains().length">
-          <table class="table table-striped table-hover">
-            <thead>
-            <tr>
-              <th>{% trans "Project Name" %}</th>
-              <th>{% trans "Last Updated" %} ({{ timezone }})</th>
-              <th></th>
-            </tr>
-            </thead>
-            <tbody data-bind="foreach: linked_domains">
-            <tr>
-              <td><a data-bind="attr: {'href': domain_link}, text: linked_domain"></a></td>
-              <td data-bind="text: last_update"></td>
-              <td>
-                <button type="button" class="btn btn-danger" data-bind="click: $root.deleteLink.bind($data)">
-                  <i class="fa fa-trash"></i>
-                  {% trans 'Delete Link' %}
-                </button>
-              </td>
-            </tr>
-            </tbody>
-          </table>
+      <div class="tab-pane fade{% if not is_linked_domain %} in active{% endif %}" id="tabs-manage-downstream">
+        <div id="ko-tabs-manage-downstream" class="ko-template">
+          <div data-bind="if: domain_links().length">
+            <table class="table table-striped table-hover" id="linked-domains-table">
+              <thead>
+              <tr>
+                <th>{% trans "Project Name" %}</th>
+                <th>{% trans "Last Updated" %} ({{ timezone }})</th>
+                <th></th>
+              </tr>
+              </thead>
+              <tbody data-bind="foreach: domain_links">
+              <tr>
+                <td><a data-bind="attr: {'href': domain_link}, text: linked_domain"></a></td>
+                <td data-bind="text: last_update"></td>
+                <td>
+                  <button type="button" class="btn btn-danger" data-bind="click: $root.deleteLink.bind($data)">
+                    <i class="fa fa-trash"></i>
+                    {% trans 'Delete Link' %}
+                  </button>
+                </td>
+              </tr>
+              </tbody>
+            </table>
+            <pagination data-apply-bindings="false"
+              params="goToPage: goToPage,
+                      perPage: itemsPerPage,
+                      totalItems: totalItems,
+                      slug: 'linked-domains-table',
```

Comment:
I think this slug is just for a cookie if you're using one, which I don't think you are.